bindings/protocols/mqtt/index.html

```diff
@@ -60,11 +60,18 @@
 
 <body>
     <section id="abstract">
-      In the context of the Web of Things, a Binding Template is 
-      a blueprint that gives guidance on how to implement a specific IoT protocol, data format or IoT platform. 
-      This document gives implementation guidelines regarding the MQTT protocol. 
-      The MQTT protocol is a lightweight, asynchronous, transport-independent, open protocol for publishing and subscribing to messages. 
-      It has been employed in many IoT platforms and IoT applications for its simplicity and scalability.
+        <p>
+            In the context of the Web of Things (WoT), a Binding Template is a blueprint that gives guidance on how to 
+            implement a specific IoT protocol, data format or IoT platform. 
+            This document gives implementation guidelines regarding the MQ Telemetry Transport (MQTT) protocol, which is
```

Comment:
If I am not mistaken, MQTT nowadays is not an abbreviation anymore. See page 2 of [these OASIS minutes](https://www.oasis-open.org/committees/download.php/49028/OASIS_MQTT_TC_minutes_25042013.pdf):

> The TC agrees that MQTT should not be standing for anything.

```suggestion
            This document gives implementation guidelines regarding the MQTT protocol, which is
```